testing/adios2/interface/TestADIOSSelection.cpp

```diff
@@ -129,6 +129,8 @@ MultiArray<T, 2> makeArray(std::initializer_list<std::initializer_list<T>> t)
     return arr;
 }
 
+std::string engine = "BPfile";
```

Comment:
const*

Reply:
This has an assignment if the test has a command line argument.